gulpfile.js

```diff
@@ -36,7 +36,7 @@ const options = {
         query: ['src/standalone.js', 'src/scripts/apploader.js']
     },
     css: {
-        query: ['src/**/*.css', 'src/**/*.scss']
+        query: ['src/**/*.css', 'src/**/*.scss', '!src/styles/**/*']
```

Comment:
Probably should filter the output, not input. When changing files in `src/styles`, they don't trigger the css watch 🤦

Reply:
I guess that I should remove this change and just use sass partial convention (`_file.scss`)